stdlib/public/Concurrency/Task.swift

```diff
@@ -122,14 +128,19 @@ extension Task {
     }
   }
 
-  /// Attempt to cancel the task.
+  /// Indicate that the task should stop.
```

Comment:
I don't think "stop" is a safe word to use here, it makes one thing we actively "stop tasks from executing" which isn't the case. It's a bit repetetive but I think we must stick to cancellation wording here, and not introduce new verbs like stopping etc.

Reply:
Maybe we can brainstorm verbs?  The guidelines for writing reference strongly discourage repeating the API symbol name in the abstract (this first line of docs), to avoid the problem of saying things like:

Reply:
I see what you mean with it then just repeating the API name... I don't have any other great ideas though :-( Stops makes me pretty worried, we never "stop" anything, the task may continue to run forever for all we know hmm...

You're right on "attempts" that'd be potentially misleading.

Reply:
Maybe... 

> Puts the task in cancelled state, for which it may or may not check and act upon.

?

Reply:
Thanks.  How about "Requests that the task should cancel its work."?  Or "Indicates that the task should stop running."?

The fact that a task might ignore cancellation doesn't need to be in the abstract; we discuss it in the paragraphs below.